src/pdn/src/pdn.tcl

```diff
@@ -376,6 +376,10 @@ proc add_pdn_stripe { args } {
   }
 
   if { [info exists flags(-followpins)] } {
+    if { [info exists keys(-starts_with)] } {
+      utl::error PDN 211 "Option -starts_with cannot be used with -followpins.\
+        Flip sites when initializing floorplan to change followpin power/ground order."
+    }
```

Comment:
I think a warning is sufficient. No need to add the "Flip sites when initializing floorplan to change followpin power/ground order."

Reply:
I remove "Flip sites..." also in the doc?

Reply:
I think it's fine there.

Reply:
sure, that would be fine